src/runtime/components/CldPicutre.vue

```diff
@@ -0,0 +1,48 @@
+<script setup lang="ts">
+import { ref } from 'vue';
```

Comment:
suggestion: Apart from the component itself it would be useful to add the documentation with examples as it is done for other components